src/site/_includes/path.njk

```diff
@@ -1,5 +1,9 @@
 ---
 layout: default
+renderData:
```

Comment:
I think renderData is deprecated in favor of using [elventyComputed](https://www.11ty.dev/docs/data-computed/)

I think you might be able to do this with eleventyComputed in this way:

```js
eleventyComputed:
  title: {{ paths[pathName].title }}
```

Reply:
done